zio-http/src/main/scala/zhttp/service/Handler.scala

```diff
@@ -244,4 +271,13 @@ private[zhttp] final case class Handler[R](
       _ <- ChannelFuture.unit(ctx.writeAndFlush(LastHttpContent.EMPTY_LAST_CONTENT))
     } yield ()
   }
+
+  private def unsafeWriteFileContent(path: java.nio.file.Path)(implicit ctx: ChannelHandlerContext) = {
+    import java.io.RandomAccessFile
+    val file       = new File(path.toString)
```

Comment:
should throw an 500 error if there is a failure. 
Add unit tests to handle error.

Reply:
Changed the method signature to address this. 
`HttpData.fromFile(file : java.io.File) = HttpData.File(file)`